lib/node_modules/@stdlib/blas/ext/base/dapxsumors/src/main.c

```diff
@@ -0,0 +1,64 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/blas/ext/base/dapxsumors.h"
+#include "stdlib/blas/base/shared.h"
+#include "stdlib/strided/base/stride2offset.h"
+
+/**
+* Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using ordinary recursive summation.
+*
+* @param N        number of indexed elements
+* @param alpha    constant
+* @param X        input array
+* @param strideX  stride length
+* @return         output value
+*/
+double API_SUFFIX(stdlib_strided_dapxsumors)( const CBLAS_INT N, const double alpha, const double *X, const CBLAS_INT strideX ) {
+	CBLAS_INT ox = stdlib_strided_stride2offset( N, strideX );
+	return API_SUFFIX(stdlib_strided_dapxsumors_ndarray)( N, alpha, X, strideX, ox );
+}
+
+/**
+* Adds a scalar constant to each double-precision floating-point strided array element and computes the sum using ordinary recursive summation and alternative indexing semantics.
+*
+* @param N        number of indexed elements
+* @param alpha    scalar
```

Comment:
```suggestion
* @param alpha    scalar constant
```

Just to be consistent for the two descriptions.